app/ports/management/commands/load-rsync.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from ports.models import Port, LastPortIndexUpdate
+from parsing_scripts import rsync_operations
+
+
+class Command(BaseCommand):
+
+    help = "Populates the database with Initial data after fetching portindex.json from rsync"
+
+    def handle(self, *args, **options):
+        if not Port.objects.count() > 0:
+            # Fetch from rsync
+            rsync_operations.sync()
+
+            # Open the file
+            data = rsync_operations.open_file()
+
+            # Start populating
+            Port.load(data['ports'])
```

Comment:
Does it make sense to use some kind of `initial` prefix? (inital_load, first_load, first_population, initial_population, ... I don't like any of my own proposals here, but I'm sure we can come up with something that's self-explanatory.)